validation.js

```diff
@@ -12,7 +12,6 @@ button.insertAdjacentElement('beforebegin', errorMessage);
 form.addEventListener('submit', (event) => {
   event.preventDefault();
 
-
   if (email.value !== email.toLocaleLowerCase()) {
     errorMessage.innerText = EMAIL_CASE_SENSITIVITY_CHECK;
   } else {
```

Comment:
You did a good job implementing some of the required changes. However, when I tried submitting with an email in upper case it submitted normally. Kindly try to fix that issue.
maybe you can use the following:
` if (email.value !== email.value.toLowerCase()) {
    invalidMessage.textContent = 'please type your whole email address in lowercase';
  } else {
    invalidMessage.textContent = '';
    form.submit();
  }`